loadtest/user/userentity/actions.go

```diff
@@ -147,13 +147,25 @@ func (ue *UserEntity) GetPostsForChannel(channelId string, page, perPage int) er
 	if resp.Error != nil {
 		return resp.Error
 	}
-	posts := make([]*model.Post, 0, len(postlist.Posts))
+	return ue.store.SetPosts(ue.postsMapToSlice(postlist.Posts))
+}
+
+func (ue *UserEntity) GetPostsBefore(channelId, postId string, page, perPage int) error {
+	postlist, resp := ue.client.GetPostsBefore(channelId, postId, page, perPage, "")
+	if resp.Error != nil {
+		return resp.Error
+	}
+	return ue.store.SetPosts(ue.postsMapToSlice(postlist.Posts))
+}
+
+func (ue *UserEntity) postsMapToSlice(postsMap map[string]*model.Post) []*model.Post {
```

Comment:
Given that we have three methods doing the same thing I think is better to extract it to a new method. I've done in a non-exported method inside userentity but maybe we can have a helper or something. What do you think?

Reply:
Yes. It shouldn't be a method of `UserEntity` but a simple function. We can make a `utils.go` inside `userentity` package.

Reply:
Done